package.json

```diff
@@ -48,7 +48,7 @@
     "@qooxdoo/eslint-config-jsdoc-disable": "^1.0.1",
     "@qooxdoo/eslint-config-qx": "1.2.1",
     "@qooxdoo/eslint-plugin-qx": "^1.2.4",
-    "@qooxdoo/framework": "^6.0.0-beta",
+    "@qooxdoo/framework": "6.0.0-beta-20200401-1413",
```

Comment:
This is unneeded - the error was a travis problem. I just restarted the Job and it works.